src/witan/models/run_models.clj

```diff
@@ -102,10 +103,26 @@
       (data-csv/write-csv out-file
                           (convert-row-maps-to-vector-of-vectors ordered-data)))))
 
+(defn get-district [gss-code]
+  (let [district-lookup (read-string (slurp (str "resources/default_datasets/local_authorities.edn")))]
```

Comment:
I'm not sure why you need `str` around the filepath. But an alternative would be to use `clojure.edn` like:
```
(clojure.edn/read-string (slurp "resources/default_datasets/local_authorities.edn"))
```